WrightTools/data.py

```diff
@@ -1934,6 +1934,62 @@ def from_text(filepath, name=None, verbose=True):
     indicies = np.arange(1)
     for i in indicies:
         axis = Axis(arr[i], 'wn', name='wm')
+        signal = Channel(arr[i + 1], name='signal', label='units', signed=False)
+        if name:
+            data = Data([axis], [signal], source='txt file', name=name)
+        else:
+            name = filepath.split('//')[-1].split('.')[0]
+            data = Data([axis], [signal], source='txt file', name=name)
+    # finish
+    if verbose:
+        print('{0} data objects successfully created from file:'.format(len(indicies)))
+    return data
+
+
+def from_BrunoldrRaman(filepath, name=None, verbose=True, trash_pixels=56):
+    """ Create a data object from plaintext tab deliminated file
+
+    Expects one energy (in wavenumbers) and one counts value.
+
+    Parameters
+    ----------
+    filepath : string, list of strings, or array of strings
+        Path to .txt file.
+    name : string (optional)
+        Name to give to the created data object. If None, filename is used.
+        Default is None.
+    verbose : boolean (optional)
+        Toggle talkback. Default is True.
+
+    Returns
+    -------
+    data
+        New data object(s).
+    """
+    if isinstance(filepath, type([])) or isinstance(filepath, type(np.array([]))):
+        return [from_BrunoldrRaman(f) for f in filepath]
+    if not os.path.isfile(filepath):
+        raise wt_exceptions.FileNotFound(path=filepath)
+    if not filepath.endswith('txt') and not filepath.endswith('dpt'):
+        wt_exceptions.WrongFileTypeWarning.warn(filepath, 'txt or dpt')
+    # import array
+    lines = []
+    with open(filepath, 'r') as f:
+        for i in range(trash_pixels):
```

Comment:
I went upstairs and talked to the Brunold group and it looks like the negative pixel problem has been either fixed or never existed. I've taken the trash_pixels out of this code and changed the example code too.